locations/spiders/elsie_sante_fr.py

```
@@ -0,0 +1,21 @@
+
+from scrapy.http import TextResponse
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class ElsieSanteFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "elsie_sante_fr"
+    item_attributes = {"brand": "Elsie Santé", "brand_wikidata": "Q127603214"}
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)